src/app.scss

```diff
@@ -1,3 +1,3 @@
-@import './components/TutorialHeader/tutorial-header.scss';
-@import './content/LandingPage/landing-page.scss';
-@import './content/RepoPage/repo-page.scss';
```

Comment:
These stylesheets are needed!

Reply:
Hi,Kelly
Thanks a lot for review my code. I correct it and commit it again.